docs/tutorial.rst

```diff
@@ -709,13 +709,17 @@ non-default limit. This reaction is an ATP maintenance reaction and the
 modelers chose to force a certain level of flux through it to simulate the
 general energy cost of cellular maintenance.
 
-Medium
-~~~~~~
+Exchange compounds
+~~~~~~~~~~~~~~~~~~
 
-The medium file is where you can designate the boundary conditions for the
-model. The compartment of the medium compounds can be designated using the
+The exchange file is where you can designate the boundary conditions for the
+model. This includes both compounds that are allowed to enter the model system
+(*the medium*) as well as compounds that are allowed to exit the model.
+Settings a negative lower limit on an exchange compounds allows an uptake of
+that compound thereby making the compound part of the medium. The
```

Comment:
This might be confusing to say making it part of the medium. This sentence "Settings a negative lower limit on an exchange compounds allows an uptake of that compound thereby making the compound part of the medium." could probably be removed to avoid the confusion of saying some subset of the exchange will be called the medium.